backend/src/test/java/com/relay/controllers/RelayControllerTest.java

```diff
@@ -16,8 +15,6 @@
 import com.relay.model.Relay;
 import com.relay.service.RelayService;
 
-import reactor.core.publisher.Flux;
-
 public class RelayControllerTest extends AbstractControllerTest {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add some tests to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2187)